_query-dsl/geo-and-xy/index.md

```diff
@@ -24,13 +24,13 @@ xy queries return documents that contain:
 
 ## Geographic queries
 
-Geographic queries search for documents that contain geospatial geometries. These geometries can be specified in [`geo_point`]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/geo-point) fields, which support points on a map, and [`geo_shape`]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/geo-shape) fields, which support points, lines, circles, and polygons. 
+Geographic queries search for documents that contain geospatial geometries. These geometries can be specified in [`geo_point`]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/geo-point/) fields, which support points on a map, and [`geo_shape`]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/geo-shape/) fields, which support points, lines, circles, and polygons. 
 
 OpenSearch provides the following geographic query types:
 
 - [**Geo-bounding box queries**]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/geo-and-xy/geo-bounding-box/): Return documents with geopoint field values that are within a bounding box. 
-- **Geodistance queries** return documents with geopoints that are within a specified distance from the provided geopoint.
-- **Geopolygon queries** return documents with geopoints that are within a polygon.
-- **Geoshape queries** return documents that contain:
+- [**Geodistance queries**]({{site.url}}{{site.baseurl}}/query-dsl/geo-and-xy/geodistance/): Return documents with geopoints that are within a specified distance from the provided geopoint.
+- **Geopolygon queries**: Return documents with geopoints that are within a polygon.
+- **Geoshape queries**: Return documents that contain:
     - geoshapes and geopoints that have one of four spatial relations to the provided shape: `INTERSECTS`, `DISJOINT`, `WITHIN`, or `CONTAINS`.
```

Comment:
Apologies for commenting outside the new content, but these list items should be initially capitalized.